saleor/static/dashboard-next/hooks/useModalDialogOpen/useModalDialogOpen.test.ts

```diff
@@ -0,0 +1,55 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import useModalDialogOpen from "./useModalDialogOpen";
+
+const onClose = jest.fn();
+const onOpen = jest.fn();
+
+const cbs = {
+  onClose,
+  onOpen
+};
+
+test("Does not render errors after close", () => {
+  const { rerender } = renderHook(
+    ({ open, cbs }) => useModalDialogOpen(open, cbs),
+    {
+      initialProps: {
+        cbs,
+        open: false
+      }
+    }
+  );
+
+  // Open modal
```

Comment:
i think it should be wrapped in something like this: `it(should open modal, () => {})`